test/lib/registration/registration.test.ts

```diff
@@ -292,16 +323,20 @@ describe("Registration", () => {
       const firstUse = json.beacons[0].uses[0];
       expect(firstUse.vhfRadio).toBe(true);
       expect(firstUse.fixedVhfRadio).toBe(true);
-      expect(firstUse.fixedVhfRadioValue).toBe("0117");
+      expect(firstUse.fixedVhfRadioValue).toBe(use.fixedVhfRadioInput);
       expect(firstUse.portableVhfRadio).toBe(true);
-      expect(firstUse.portableVhfRadioValue).toBe("0118");
+      expect(firstUse.portableVhfRadioValue).toBe(use.portableVhfRadioInput);
       expect(firstUse.satelliteTelephone).toBe(true);
-      expect(firstUse.satelliteTelephoneValue).toBe("0119");
+      expect(firstUse.satelliteTelephoneValue).toBe(
+        use.satelliteTelephoneInput
+      );
       expect(firstUse.mobileTelephone).toBe(true);
-      expect(firstUse.mobileTelephone1).toBe("01178123456");
-      expect(firstUse.mobileTelephone2).toBe("01178123457");
+      expect(firstUse.mobileTelephone1).toBe(use.mobileTelephoneInput1);
+      expect(firstUse.mobileTelephone2).toBe(use.mobileTelephoneInput2);
       expect(firstUse.otherCommunication).toBe(true);
-      expect(firstUse.otherCommunicationValue).toBe("Via email");
+      expect(firstUse.otherCommunicationValue).toBe(
+        use.otherCommunicationInput
+      );
```

Comment:
Partial replacement of literal values with mock properties.  I didn't replace the booleans because I didn't want to spend too much time / grow the PR on playing with out-of-scope tests.